DataStructure/src/main/list/ResizableArray.java

```diff
@@ -23,18 +23,23 @@ public void Add(int e){
     public int Get(int index){
         return list[index];
     }
-
-    public void Remove(){
+    public void Set(int index, int e) {
+     list[index] = e;
+    }
+    public int Remove(){
         --i;
         System.out.println("Removed : "+list[i]);
+        return list[i];
     }
 
-    public void Find(int f){
+    public int Find(int f){
         for (int j = 0; j < list.length; j++) {
             if(f == list[j]){
                 System.out.println("Element : "+f+"\n Found at index : "+j);
-                break;
+                return j;
             }
         }
+        System.out.println("Element Not Found");
+        return 0;
```

Comment:
Is it fine to return 0 as it may consider it index 0. What I can think of is to return string by converting the value of j in the for loop to string. And if it is not found than returning a statement.